### PR TITLE
Fix: comment responsive logic

### PR DESCRIPTION
**Pull request by tokitouq**

### CODE REVIEW

File: backend/django_core/apps/user/migrations/0010_alter_customuser_username.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+                    apps.user.validators.username.username_validator,
+                    django.core.validators.RegexValidator(
+                        "^[a-zA-Z0-9_-]+#[0-9]{4}$",
+                        message="Username is not valid for this regex `^[a-zA-Z0-9_-]+#[0-9]{4}$`",
+                    ),
+                ],
+                verbose_name="username",
+            ),
+        ),
+    ]
```
Comment (baseplate-admin) [inline backend/django_core/apps/user/migrations/0010_alter_customuser_username.py:33]: This is a local file, it should not be comitted

File: backend/django_core/static_src/components/minor/Comment/ReplyButton.svelte [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    <span>Replay</span>
+</button>
+
+<button
+    on:click={() => handle_click(DeviceEnum.Desktop)}
+    class={cn(`btn h-max min-h-full !bg-transparent p-0 text-xs md:gap-[0.35vw] md:text-[0.9vw] hidden md:flex`)}
+>
+    <Chat class="w-4 md:w-[1vw]" />
+    <span>Replay</span>
+</button>
```
Comment (baseplate-admin) [inline backend/django_core/static_src/components/minor/Comment/ReplyButton.svelte:36]: This is not a good way to handle this

Comment (tokitouq) [reply backend/django_core/static_src/components/minor/Comment/ReplyButton.svelte]: What about this logic: https://github.com/coreproject-moe/CoreProject-Monorepo/pull/3602/commits/10d62461a28caac2a000691fafaa2ed82b3f0d4c

File: backend/django_core/apps/user/migrations/0010_alter_customuser_username.py (hunk 1 above)
Comment (tokitouq) [reply backend/django_core/apps/user/migrations/0010_alter_customuser_username.py:33]: https://github.com/coreproject-moe/CoreProject-Monorepo/blob/fix/comment-responsive-logic/backend/django_core/apps/user/migrations/0010_alter_customuser_username.py
This is committed by you.

See migrations are being comment.

Comment (baseplate-admin) [reply backend/django_core/apps/user/migrations/0010_alter_customuser_username.py:33]: Fun fact is i dont see this migration on my system lol

File: backend/django_core/static_src/components/minor/Comment/ReplyButton.svelte (hunk 2 above)
Comment (baseplate-admin) [reply backend/django_core/static_src/components/minor/Comment/ReplyButton.svelte]: Even worse

File: backend/django_core/static_src/components/minor/Comment/CommentBlock.svelte [hunk 3]
```diff
@@ -136,6 +125,30 @@
...
+                toggle_reply_shown();
+            };
+        },
+        handle_desktop_click = () => {
+            if (item.depth > 4) {
+                return; // link
+            } else {
+                toggle_reply_shown();
+            };
+        };
```
Comment (baseplate-admin) [inline backend/django_core/static_src/components/minor/Comment/CommentBlock.svelte:151]: waujt

**Pull request merged**